packages/@react-spectrum/breadcrumbs/src/BreadcrumbItem.tsx

```diff
@@ -52,13 +55,27 @@ export function BreadcrumbItem(props: BreadcrumbItemProps) {
         )
     }
   );
+  let provider = useProvider();
 
   return (
     <Fragment>
       <FocusRing focusRingClass={classNames(styles, 'focus-ring')}>
         {element}
       </FocusRing>
-      {isCurrent === false &&
+      {isCurrent === false && (provider?.theme?.global?.express ? (
```

Comment:
Don't the icons themselves usually do this swapping? I couldn't find another component where we do this, except in generateIcons.cjs.

Reply:
Correct, but it's only in ui icons. This is a UI icon in spectrum, but a workflow icon in express. I didn't want to muddy the waters by copying it into the automated swapping script because the chevron right is used in more places than just here, and I didn't want to affect every other component. If the others all end up needing this chevron instead, we can swap wholesale.

Reply:
ok I guess. Really didn't want to start changing things in code for specific themes...

Reply:
yeah, definitely not ideal
I did a quick search of all the other instances of chevrons in the express designs, they all use the 18x18. I'll try swapping it out in the same manner as the others and will check in chromatic to see how that behaves.

Reply:
did the swapping ChevronRight in the script break stuff in chromatic? Just checking before I approve since everything else looks good to me but it would def be great if we didn't need to check the theme here